i18n/locales/source/hourofcode/en.yml

```diff
@@ -30,6 +30,14 @@
   oceans_landing_resource_button: 'View Resources'
   oceans_landing_cs_for_good: 'CS For Good'
   oceans_landing_cs_for_good_desc: 'Resources to help students understand the role computer science could play in creating a more equitable world.'
+  oceans_landing_learn_more: 'Learn more about AI and Machine Learning'
+
+  ai_video_title_what_is_ai: 'What is AI?'
+  ai_video_title_phone_knows_dog: 'How does your phone know this is a dog?'
+  ai_video_title_what_is_ml: 'What is Machine Learning?'
+  ai_video_title_ml_big_small_prickly: 'Machine Learning: Solving Problems Big, Small and Prickly'
+  ai_video_title_ethics_of_ai: 'Ethics of AI'
+  ai_video_title_ml_human_bias: 'Machine Learning and Human Bias'
```

Comment:
these changes should actually be made to https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/hourofcode.com/i18n/en.yml